platform/javascript/export/export_plugin.cpp

```diff
@@ -142,12 +143,18 @@ void EditorExportPlatformJavaScript::_fix_html(Vector<uint8_t> &p_html, const Re
 		head_include += "<script type='application/javascript'>window.addEventListener('load', () => {if ('serviceWorker' in navigator) {navigator.serviceWorker.register('" +
 				p_name + ".service.worker.js');}});</script>\n";
 	}
-
+	else {
+		const String date = Time::get_singleton()->get_date_string_from_system();
+		const String time = Time::get_singleton()->get_time_string_from_system();
+		const uint64_t ms = Time::get_singleton()->get_ticks_msec() % 1000;
```

Comment:
That's not the clock time in ms, that's the amount of time since the editor process was started. It's pretty arbitrary so I don't think it's relevant to export.

Reply:
Thank you for your comment. Yes I agree it is not the clock time in milliseconds. But that does not matter for a cache buster. It does not have to be a valid time. It just needs to be unique within the second. (this is just in case that an automated UI unit tester presses two times on the button within a second and compares the result)

Reply:
As an aside, it would be great to have a UUID in godot core...  It's probably useful in a whole bunch of places in the engine.